spec/System/HapistranoSpec.hs

```diff
@@ -40,7 +40,7 @@ genSourceRepo path = do
         , "echo testing > " ++ joinPath [clonePath, "README"]
         , "cd " ++ clonePath ++ " && git add -A"
         , "cd " ++ clonePath ++ " && git commit -m\"First commit\""
-        , "cd " ++ clonePath ++ " && git push"
+        , "cd " ++ clonePath ++ " && git push origin master"
```

Comment:
@jsl Is `"origin master"` correct? `git push` fails in my computer unless I specify what default behavior I want. I don't know if that's the case with the Git installed in Travis.